_stable/download-scripts.js

```diff
@@ -0,0 +1,108 @@
+/** @param {NS} ns **/
+export async function main(ns) {
+  await ns.wget(
+    "https://raw.githubusercontent.com/Jacques-Murray/bitburner-automation/main/_stable/ap-hacknet-node.js",
```

Comment:
Very nice. This is a great utility tool. Do we want to point it to `chrisrabe` instead of `Jacques-Murray` user because this repo would have contributions from other people too?

Reply:
I updated this already with a revised version where a list is used in a for loop to allow for easier update of additional scripts